### PR TITLE
Update About page

### LINKED ISSUE #779: Add more information to the About page
Currently the About page on the web app describes an overview of the project, the technologies used, an overview of the instrument monitoring, and a description of the development team.

There are number of things we could add to this page to more thoroughly describe the project and how users may interact with it.

Here is a running list of things to add (or point to existing documentation via a hyperlink):
- [x] Describe the process for submitting a bug report and/or feature request (See #788)
- [x] Describe the process for authentication
- [x] Describe the process for contributing software changes to the project
- [x] Describe which data the application serves and where it comes from
- [x] Describe which monitors we support and where the algorithms come from
- [x] Add a table of contents

### PR DESCRIPTION
**Pull request by bourque**
Closes #779

### CODE REVIEW

**Comment by bourque** [comment]
@bhilbert4 @bsunnquist @shanosborne @mfixstsci Could I bother one of you to review this one for me?  I reckon we are all quite busy but if someone can spare a few minutes to look this over it would be much appreciated!  Thank you.

**Comment by bsunnquist** [comment]
@bourque I can review it

File: jwql/website/apps/jwql/templates/about.html [hunk 1]
```diff
@@ -9,46 +9,144 @@
...
             <img src={{ static('img/jwql_instrument_monitors.png') }}
                  width=100%
                  alt="JWQL instrument monitors"
                  title="">
+        </div>
+
+
+        <h4 id="engineering_database">Engineering Database</h4><br>
+
+        The JWQL application allows users to query the Engineering Database (EDB).  Users may search of mnemonics and plot a particular mnemonic over a given time period.  For more information about the EDB, see (some page)
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:91]: "search for" mnemonics

Some possible links to include in the "some page" placeholder:
https://outerspace.stsci.edu/display/MASTDOCS/Engineering+Data
https://outerspace.stsci.edu/display/MASTDOCS/Using+the+Engineering+Data+Portal

File: jwql/website/apps/jwql/templates/about.html [hunk 2]
```diff
@@ -9,46 +9,144 @@
...
+                 alt="programmatic API example"
+                 title="">
+        </div>
+
+
+
+        <h2 id="access_and_authentication">Access and Authentication</h2><br>
+
+
+        Due to the presence of restricted JWST data, access to the JWQL web app is currently restricted to JWST instrument team members at STScI, JWQL developers, and other select individuals from other internal teams (e.g. DMS). Access is controlled via a hard-coded Active Directory (AD) groups named. Individuals may be added or removed from these groups by submitting a request to <a href='mailto: jwql@stsci.edu'>jwql@stsci.edu</a><br><br>
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:130]: "AD groups named" - maybe typo?

File: jwql/website/apps/jwql/templates/about.html [hunk 3]
```diff
@@ -58,11 +156,22 @@ <h4>The Development Team</h4>
...
 
-        <br>
-        <h4>Contact</h4>
-        To submit a bug report or suggest a new feature, please <a href="https://github.com/spacetelescope/jwql/issues" target="_blank">create a new issue</a> on our GitHub repository. The JWQL team will prioritize the issue once it is received. Any other questions, comments, or concerns can be directed to <a href='jwql@stsci.edu'>jwql@stsci.edu</a>.<br><br>
 
-        <h4>Acknowledgements</h4>
+
+        <h2 id="contributing">Contributing</h2><br>
+
+        The JWQL development team welcomes contributions from anyone at STScI.  If you are interested, please see the <a href="https://github.com/spacetelescope/jwql#software-contributions" target="_blank">Software Contirubtions</a> documentation.<br><br>
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:163]: contributions typo

File: jwql/website/apps/jwql/templates/about.html [hunk 4]
```diff
@@ -9,46 +9,144 @@
...
+            <img src={{ static('img/edb.png') }}
+                 width=100%
+                 alt="Engineering Database example"
+                 title="">
+        </div>
+
+
+        <h4 id="jwqldb_database">JWQLDB Database</h4><br>
+
+        Instrument monitoring results and other metadata get stored in a database called the "JWQLDB" database.  Users may view the contents of this database directly through the browser.
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:103]: may be worth linking to our JWQLDB webpage here

File: jwql/website/apps/jwql/templates/about.html (hunk 1 above)
Comment (bsunnquist) [reply jwql/website/apps/jwql/templates/about.html:91]: also may be worth turning "Engineering Database" into a link to our EDB webpage

File: jwql/website/apps/jwql/templates/about.html [hunk 5]
```diff
@@ -9,46 +9,144 @@
...
+                 width=100%
+                 alt="JWQLDB example"
+                 title="">
+        </div>
+
+
+        <h4 id="programmatic_api">Programmatic API</h4><br>
+
+
+        The JWQL application allows users to gather various JWST archival metadata through a programmatic API.
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:116]: likewise, may be worth turning "programmatic API" into a link to our API webpage

File: jwql/website/apps/jwql/templates/about.html [hunk 6]
```diff
@@ -9,46 +9,144 @@
...
+
+
+
+
+        <h2 id="features">Features</h2><br>
+
+
+        <h4 id="visual_inspection">Visual Inspection</h4><br>
+
+        The JWQL application offers users a way to visually inspect JWST data that is present in the MAST archive.  Users may choose to view data for a particular instrument, program, and observation.  Users may also flag images as having one or more 'anomalies'.  Flagged anomalies are then stored in the JWQLDB database for future analysis.  Additionaly, users may view the contents of the FITS header for the given obeservation, as well as visually inspect other available filetypes.
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:58]: Additionaly, obeservation typos

File: jwql/website/apps/jwql/templates/about.html [hunk 7]
```diff
@@ -72,10 +181,13 @@ <h4>Acknowledgements</h4>
         {% endfor %}
         </ul>
 
-        <h4>Other attributions:</h4>
+
+
+
+        <h4>Other attributions:</h4><br>
         <ul>
             <li>This website aims to be accessibile via 508 complience.
```
Comment (bsunnquist) [inline jwql/website/apps/jwql/templates/about.html:189]: accessibile, complience typos

**Review by bsunnquist — COMMENTED** [review]
@bourque Looks really good! Just some minor suggestions included

**Comment by bourque** [comment]
Thanks @bsunnquist!  Sorry that I missed so many typos 😅

**Comment by bourque** [comment]
@bsunnquist Ok, I think I addressed all of your suggestions

**Pull request merged**